app/scripts/views/complete_sign_up.js

```diff
@@ -44,6 +45,10 @@ function (Cocktail, FormView, BaseView, CompleteSignUpTemplate,
       // cache the email in case we need to attempt to resend the
       // verification link
       this._email = this._account.get('email');
+      this._able = options.able;
```

Comment:
If this._able is imported in [base.js](https://github.com/mozilla/fxa-content-server/pull/2943/files#diff-af500bac5338814fda67ec4de209ac46R101), no need to do it here.

An alternative is to add an `initialize` function to the experiment-mixin that saves a reference to able from within the mixin. That way Views don't necessarily have to worry about importing able, just add the mixin and away it goes?